### PR TITLE
init: sfml window

### PR DESCRIPTION
**Pull request by AxelF44**
I've init the sfml display with the class renderer who have an exception class inside ! I've added the Makefile and main file !

### CODE REVIEW

**Review by mathieurobert1 — DISMISSED** [review]
Perfect !

File: include/Renderer.hpp [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        void renderClicker();
+
+        class RendererException : public std::exception {
+            public:
+                /*
+                 * @brief Exception to throw when there is an error in renderer.
+                 * @param msg message to throw
+                 */ 
+                explicit RendererException(const std::string &msg) : message(msg) {}
+
```
Comment (Njord201) [inline include/Renderer.hpp:50]:
```suggestion
                 * @brief Exception to throw when there is an error in renderer.
                 *
                 * @param msg message to throw
                 */
                explicit RendererException(const std::string &msg) : message(msg) {}

```

File: include/Renderer.hpp [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        class RendererException : public std::exception {
+            public:
+                /*
+                 * @brief Exception to throw when there is an error in renderer.
+                 * @param msg message to throw
+                 */ 
+                explicit RendererException(const std::string &msg) : message(msg) {}
+                
+                /* @brief Return the message of error.
+                 * @return message
```
Comment (Njord201) [inline include/Renderer.hpp:52]:
```suggestion
                /* @brief Return the message of error.
                 *
                 * @return message
```

File: include/Renderer.hpp [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+#include <vector>
+#include <memory>
+#include <string>
+#include <iostream>
+#include <SFML/Graphics.hpp>
+
+namespace FlameCLicker
+{
+   class Renderer;
```
Comment (Njord201) [inline include/Renderer.hpp:18]: Describe the class FlameClicker

File: include/Renderer.hpp [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+#pragma once
+
+#include <vector>
+#include <memory>
+#include <string>
+#include <iostream>
+#include <SFML/Graphics.hpp>
+
+namespace FlameCLicker
```
Comment (Njord201) [inline include/Renderer.hpp:16]:
```suggestion
namespace FlameClicker
```

File: include/Renderer.hpp [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+         */
+        ~Renderer() = default;
+
+        /**
+         * @brief Render the clicker in a window with SFML.
+         *
+         */
+        void renderClicker();
+
+        class RendererException : public std::exception {
```
Comment (Njord201) [inline include/Renderer.hpp:43]: Try to describe the exception class in the cpp file

File: src/Renderer.cpp [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+void FlameCLicker::Renderer::renderClicker() {
+    while (_window.isOpen()) {
+        sf::Event event;
+        while (_window.pollEvent(event)) {
+            if (event.type == sf::Event::Closed || (event.type == sf::Event::KeyPressed && event.key.code == sf::Keyboard::Escape))
+                _window.close();
+        }
+
+        _window.clear();
+        // Draw
```
Comment (Njord201) [inline src/Renderer.cpp:22]: Create an issue

**Review by Njord201 — CHANGES_REQUESTED** [review]

**Review by mathieurobert1 — APPROVED** [review]

**Review by theophile-jr — DISMISSED** [review]

**Review by mathieurobert1 — APPROVED** [review]

**Review by theophile-jr — APPROVED** [review]

**Review by Njord201 — APPROVED** [review]
Good Job !

**Pull request merged**